client/blocks/stats-sparkline/style.scss

```diff
@@ -0,0 +1,16 @@
+.stats-sparkline {
+	padding: 5px 0 2px;
+}
+
+.stats-sparkline__bar {
+	display: inline-block;
+	background-color: var( --color-sidebar-gridicon-fill );
+	width: 1px;
+	height: 100%;
+	transform-origin: 0% 100%;
+	margin-left: 1px;
+
```

Comment:
Can we also add a `.selected &` or `.sidebar__menu-item-parent.selected &` with the `color: var(--color-sidebar-menu-selected-text)` just so the color works well when the stats item is selected?

Reply:
Yep, added in the latest commit :+1: